build_debian.sh

```diff
@@ -312,9 +312,13 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     python-pip              \
     python3-pip             \
     cron                    \
-    haveged
-
+    haveged                 
 
+## Secure boot signed shim grub 
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install      \
+    grub-efi-amd64-signed           \
+    shim-signed         
```

Comment:
I am worried about the disk space of image. Are they needed in runtime? Is there anything we can do to reduce image sizes before installation, or occupied disk space after installation?